samples/smoketest/README.md

```diff
@@ -53,15 +53,29 @@ export COSMOS_KEY=""
 ```
 
 ### Running the console app
-[Python](https://www.python.org/downloads/) version 3.7.4 was used to run this sample.
+[Python](https://www.python.org/downloads/) version 2.7.16 and 3.7.4 were used to run this sample.
+
+Install the libraries required using pip:
+```
+pip install -r requiriments.txt
+```
 
 In the \SmokeTest\ directory, run Program.py
 ```
 python .\Program.py
 ```
 
-## Key concepts
+#### Important: To run the async tests
+In order to run the samples with the asynchronous clients, **python 3.5 or greater** is needed.
 
+Install the async libraries using pip:
```

Comment:
This should probably include "Install both requirements.txt and requirements_async.txt"